lib/config_cat/config_fetcher.ex

```diff
@@ -34,14 +43,49 @@ defmodule ConfigCat.CacheControlConfigFetcher do
 
     initial_state =
       default_options()
-      |> Keyword.merge(options)
-      |> Enum.into(%{})
+      |> custom_server_options(options)
+      |> Map.merge(Enum.into(options, %{}))
+      |> data_governance_options(options)
       |> Map.put(:etag, nil)
 
     GenServer.start_link(__MODULE__, initial_state, name: name)
   end
 
-  defp default_options, do: [api: ConfigCat.API, base_url: Constants.base_url()]
+  defp default_options,
+    do: %{
+      api: ConfigCat.API,
+      base_url: Constants.base_url_global(),
+      data_governance: DataGovernance.global(),
+      custom_endpoint: false,
```

Comment:
Should we name this `custom_endpoint?` to make it clearer that this is a boolean?

Reply:
Sure!